modules/Category/index.ts

```diff
@@ -1 +1,3 @@
-export { default } from './Category';
+import { Category } from './Category';
+
+export default Category;
```

Comment:
I have opted for this approach when using the component with dynamic import. This way we can still use named exports in the component itself and only need to use default export in the index file.

Reply:
We can still use the named exports for everything including components and only use the default export here for dynamically loaded modules.

Maybe we can just add comments to all the `export default` usages to clarify? 🤔

Reply:
Yes, that would be perfect!

Reply:
I've added the comments and also re-enabled the `no-default-export` rule.